enterprise_gateway/services/kernels/remotemanager.py

```diff
@@ -504,7 +562,10 @@ async def restart_kernel(self, now=False, **kwargs):
             Any options specified here will overwrite those used to launch the
             kernel.
         """
-        self.restarting = True
+        # this is added for only auto-restarter as it directly call this method.
+        self.log.info(f"restarting kernel with value for now: {now}")
+        if now:
+            self.restarting = True
```

Comment:
I don't think this logging is that helpful.  When possible, I'd rather have the comment specific to the action. Where it is makes it tough for someone reading the code know what "this is added" is referring to.  They might think its the larger block of code.
```suggestion
        if now:  # if auto-restarting (when now is True), indicate we're restarting.
            self.restarting = True
```

Reply:
yes. agree kevin.